jacovid/featurize.py

```diff
@@ -0,0 +1,151 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..')) 
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+
+def featurize_data(data, target_var="inc_hosp", h=1, feature_parameters = 
+    [{
+        "fun": 'moving_average',
+        "args": {'target_var': "inc_hosp", 'num_days': 7}
+        },
+        {
+        "fun": 'lagged_values',
+        "args": {'target_var': "inc_hosp", 'window_size': 2}
+        },
+    ]):
+    """
+    Convert data to tensors containing features x and responses y for each location
+    and time.
+
+    Parameters
+    ----------
+    data: data frame 
+        It has columns location, date, and a column with the response variable to forecast.
+        This data frame needs to be sorted by location and date columns in ascending order.
+    target_var: string
+        Name of the column in the data frame with the forecast target variable.
+        Default to "inc_hosp"
+    h: integer
+        Forecast horizon. Default to 1.
+    feature_parameters: array of dictionaries
```

Comment:
suggest renaming this to just "features", to try to keep names short
```suggestion
    features: list of dictionaries
```